graphql/src/schema/api.rs

```diff
@@ -240,6 +242,27 @@ fn add_block_height_type(schema: &mut Document) {
     schema.definitions.push(def);
 }
 
+/// Adds a global `BlockChangedFilter` type to the schema. The `block` argument
+/// accepts values of this type
+fn add_changed_block_filter_input_type(schema: &mut Document) {
```

Comment:
This is not something you need to change, but directionally, I don't like how we create all these fixed types in code, just because it makes understanding what gets added much harder. With `meta.graphql` I started defining these types in GraphQL text, which I think is much easier to read than constructing an AST in code.

If we ever make bigger changes to the API schema in the future, I think it would be great to rename `meta.graphql` and add to that whatever can be added with fixed GraphQL text.

Reply:
Yeah, I agree that the SDL in `meta.graphql` is more readable. 
I will move the definition of `BlockChangedFilter` and `Block_height`.